keras/preprocessing/image.py

```diff
@@ -580,6 +580,8 @@ def standardize(self, x):
         # Returns
             The inputs, normalized.
         """
+        if self.preprocessing_function:
```

Comment:
@Dref360 Minor typo. This needs to be:
```
if self.preprocessing_function:
      x = self.preprocessing_function(x)
```

Reply:
@Dref360 Might be also worth changing the documentation of preprocessing_function from "The function will run before any other modification on it." to "The function will run before any other standardization is applied." or something similar. :)

Reply:
My bad sorry :'( We didn't have any test for that (weird...) So I added a test, will get merged shortly.

Reply:
As for the doc, We'll handle it later.

Reply:
sounds good thanks!